.github/workflows/deploy.yml

```diff
@@ -157,8 +157,6 @@ jobs:
       # Install the latest version of Terraform CLI
       - name: Setup Terraform
         uses: hashicorp/setup-terraform@v2
-        with:
-          terraform_version: 1.4.7
```

Comment:
I'd rather update the version here to avoid further unexpected inconsistencies between ovh and terraform

Reply:
version fixed to 1.6.X